docker/Dockerfile-bionic

```diff
@@ -0,0 +1,46 @@
+# -*- mode: dockerfile -*-
```

Comment:
Can we update Dockerfile-xenial in the same way?

Reply:
Should already be there :)